### ISSUE TITLE
Finding Convex Hull problem

### ISSUE THREAD

**Issue by itnug987** [jainaman224/Algo_Ds_Notes#564]
I want to add Jarvis's algorithm and the Graham Scan algorithm to find the smallest convex polygon containing all the given points in Java.

**Comment by rishabhgarg25699** [comment]
Yes, You can go ahead with the Jarvis's algorithm and the Graham Scan algorithm. These are the good algorithms for finding the smallest convex polygon that contains all the points . But please comment the code carefully so that the reader of the code can easily understand the code as these algos are typical to understand fully.

**Comment by itnug987** [comment]
Yeah sure. Thank you.

**Comment by sez98** [comment]
i would like to do the same in c++ and python? please assign me

**Comment by jainaman224** [comment]
@itnug987 Are you working on it. If yes then please specify language and algorithm.

**Comment by jainaman224** [comment]
@sez98 I will update you once @itnug987 replies.

**Comment by itnug987** [comment]
@jainaman224 Yes I am working on it. Algorithms: Jarvis's algorithm and the Graham Scan algorithm. Language: Java

**Comment by sez98** [comment]
@jainaman224 can do the same in language: c++ and python?

**Comment by jainaman224** [comment]
@sez98 Go ahead.

**Comment by jainaman224** [comment]
@sez98 Are you working on it?

**Comment by jainaman224** [comment]
@itnug987 Are you working on it?

**Comment by itnug987** [comment]
@jainaman224 Yes, will make PR in next few days.

**Comment by sez98** [comment]
yes ... i am.. i ll complete it by Wednesday.

**Comment by shreyasrivastava936** [comment]
Hi! I want to work on this issue in Java. please assign this task to me for GSSoC'20.

**Comment by mugdhajoshi** [comment]
I want to work on this issue in Python as part of GSSoC 2020. Please assign it to me.

**Comment by jainaman224** [comment]
@mugdhajoshi Go ahead.

**Comment by m-code12** [comment]
Is this resolved .If no one is working on it , please assign it to me @jainaman224

**Comment by RiaJha02** [comment]
I would like to implement this problem in java and c++, please assign this task to me
Thank you!
